.clang-format

```diff
@@ -18,8 +20,8 @@ AlwaysBreakAfterDefinitionReturnType: None
 AlwaysBreakAfterReturnType: None
 AlwaysBreakBeforeMultilineStrings: false
 AlwaysBreakTemplateDeclarations: Yes
-BinPackArguments: true
-BinPackParameters: true
+BinPackArguments: false
+BinPackParameters: false
```

Comment:
Why are you including .clang-format changes along with your PR?

In any case I would not make the highlighted change. There are good reasons for using different argument lists in different situations.

Reply:
I am including these changes since I am using clang-format for development and was not satisfied with our current configuration.
Since there currently is no enforcement of these formatting rules, everyone is still free to format their code as they like so these changes are not forcing anyone to format their code in a specific way.
The reason I made these changes is that I find a long argument list much easier to read if every argument is on it's own line.

Reply:
The thing is, it is possible to have long argument lists with long variable/type names, or alternatively long argument lists with many arguments.  The first works well with each parameter on its own line, the second does not.